src/intelligence_layer/examples/qa/multiple_chunk_qa.py

```diff
@@ -29,11 +29,13 @@ class MultipleChunkQaInput(BaseModel):
             Can be arbitrarily long list of chunks.
         question: The question that will be answered based on the chunks.
         language: The desired language of the answer. ISO 619 str with language e.g. en, fr, etc.
+        explainability_enabled: Whether to generate highlights (using the explainability feature) for the answer;  default False for performance reasons
     """
 
     chunks: Sequence[TextChunk]
     question: str
     language: Language = Language("en")
+    explainability_enabled: bool = False
```

Comment:
While i think this would be a sensible default if we would introduce this feature new, we already have this functionality. To avoid user annoyance, i would default this to `True`

Reply:
All right, let's make the default  True -> [0b007b18a660a2c5dafaf7b99741141508feb34e](https://github.com/Aleph-Alpha/intelligence-layer-sdk/commit/0b007b18a660a2c5dafaf7b99741141508feb34e)